tests/System.Text.Primitives.Tests/Formatting/TimeTests.cs

```diff
@@ -51,6 +53,7 @@ public void RandomDateTimeTests(char format)
         static void TestDateTimeFormat(DateTime dt, char format, SymbolTable symbolTable)
         {
             var expected = dt.ToString(format.ToString(), CultureInfo.InvariantCulture);
+            if (format == 'r') expected = expected.ToLowerInvariant();
```

Comment:
From that link, it seems to want it both in regular RFC1123 (for **x-ms-date** header) and lowercase for **Authorization** header. So might be better to lowercase the string rather than generate it from date twice?